api/desecapi/authentication.py

```diff
@@ -14,6 +14,16 @@
 from desecapi.serializers import AuthenticatedBasicUserActionSerializer, EmailPasswordSerializer
 
 
+class DynAuthenticationMixin:
+    def authenticate_credentials(self, username, key):
+        user, token = TokenAuthentication().authenticate_credentials(key)
+        if not user.is_active:
```

Comment:
This is redundant (otherwise inactive users could authenticate against `TokenAuthentication`). `TokenAuthentication` inherits from `RestFrameworkTokenAuthentication`, where `authenticate_credentials` [unconditionally checks if the user of the given token is active](https://github.com/encode/django-rest-framework/blob/1ec0f86b585cd87e4b413aeaad1ecc947bacfef2/rest_framework/authentication.py#L205).

Reply:
done